Source/WebCore/css/CSSPseudoSelectors.json

```diff
@@ -232,6 +232,162 @@
             "settings-flag": "imageControlsEnabled",
             "user-agent-part": true
         },
+        "-webkit-caps-lock-indicator": {
+            "status": "non-standard",
+            "user-agent-part": true
+        },
+        "-webkit-color-swatch": {
+            "status": "non-standard",
+            "user-agent-part": true
+        },
+        "-webkit-color-swatch-wrapper": {
+            "status": "non-standard",
+            "user-agent-part": true
+        },
+        "-webkit-contacts-auto-fill-button": {
+            "status": "non-standard",
+            "user-agent-part": true
+        },
+        "-webkit-credentials-auto-fill-button": {
+            "status": "non-standard",
+            "user-agent-part": true
+        },
+        "-webkit-credit-card-auto-fill-button": {
+            "status": "non-standard",
+            "user-agent-part": true
+        },
+        "-webkit-date-and-time-value": {
+            "status": "non-standard",
+            "user-agent-part": true,
+            "condition": "ENABLE(DATE_AND_TIME_INPUT_TYPES)"
```

Comment:
nit: Here and below, can you move up the `condition` field to preserve alphabetic sorting?